assets/samplesheet.csv

```diff
@@ -1,5 +1,6 @@
-sample,family,cram,crai,bed,small_variants
-PosCon1,family1,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon1.cram,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon1.cram.crai,s3://test-data/genomics/homo_sapiens/illumina/regions/SVcontrol/PosCon1and2.roi.bed,s3://test-data/genomics/homo_sapiens/illumina/vcf/SVcontrol/family1_small_variants.vcf.gz
-PosCon2,family1,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon2.cram,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon2.cram.crai,s3://test-data/genomics/homo_sapiens/illumina/regions/SVcontrol/PosCon1and2.roi.bed,
-PosCon3,,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon3.cram,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon3.cram.crai,,s3://test-data/genomics/homo_sapiens/illumina/vcf/SVcontrol/PosCon3_small_variants.vcf.gz
-PosCon4,,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon4.cram,s3://test-data/genomics/homo_sapiens/illumina/cram/SVcontrol/small/PosCon4.cram.crai,s3://test-data/genomics/homo_sapiens/illumina/regions/SVcontrol/PosCon4.roi.bed,s3://test-data/genomics/homo_sapiens/illumina/vcf/SVcontrol/PosCon4_small_variants.vcf.gz
+sample,cram,crai,small_variants,gender
```

Comment:
Would it not make sense to add family as well? For trio analysis for example?

Reply:
The pipeline currently doesn't support that :) but it would be good to have in the future